drivers/wifi/siwx91x/Kconfig.siwx91x

```diff
@@ -174,6 +174,13 @@ config WIFI_SILABS_SIWX91X_ENABLE_INSTANT_SCAN
 	  can perform an advanced scan without waiting for the periodic
 	  scan interval, enabling quicker detection of available networks.
 
+config WIFI_SILABS_SIWX91X_FIRMWARE_UPDATE
```

Comment:
I believe this component does not depends on `WIFI_SILABS_SIWX91X`. `WISECONNECT_NETWORK_STACK` should be sufficient, no?